jac-uk/admin
# Fix memberships display and edit...again

---
## What's included?
A few missed checks on the objects being displayed in the application view meant that the whole section wasn't being rendered if there was an error

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Navigate to an application view
- Observe memberships are displayed and editable

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
